config/routes.rb

```diff
@@ -128,9 +128,10 @@
   end
 
   scope :user do
-    get 'settings' => "users#settings"
-    patch 'settings' => "users#update_settings"
+    get 'settings' => "settings#edit"
+    patch 'settings' => "settings#update"
   end
```

Comment:
A shorter alternative is to use a [singular resource](http://guides.rubyonrails.org/routing.html#singular-resources) here, with only the edit and update actions